handbook/people-ops/travel.md

```diff
@@ -1,98 +1,166 @@
+## [](https://about.sourcegraph.com/handbook/people-ops/travel#how-we-came-to-this-decision "#how-we-came-to-this-decision")How we came to this decision:
 
-***For any questions, clarifications, or notifications around trips you'll be taking, please email **people-ops@sourcegraph.com**.*** 
+Our policy until now has been to suspend travel/meetup reimbursement and for managers to not organize team events.  _Why not a blanket “no meetup” policy?_  On principle, we don’t think any company can or should forbid people from meeting up on their personal time. Although Covid-19 continues to deeply affect each person and region, we needed to establish a policy to meet the team’s needs. 82% of team members surveyed (49 respondents) said they’d feel comfortable attending an in-person meetup (_either now or within the next few months, with some conditions_), and 84% of team members said Sourcegraph should start allowing in-person team meetups and travel. With most team members in favor of starting to travel, we didn’t want  _only_  local meetups to start, because that wouldn’t be inclusive to people who don’t live near other team members. We decided to change the policy to start allowing travel/meetup reimbursement.
 
+***For any questions, clarifications, or approvals around trips you’ll be taking, please email  **people-ops@sourcegraph.com**.***
 
-## Team travel budgets
+# [](https://about.sourcegraph.com/handbook/people-ops/travel#team-travel-budgets "#team-travel-budgets")Travel budgets
 
-Sourcegraph is an [all-remote](../company/remote/index.md) company. This means we all need to make an effort to travel to meet up in person and spend quality time with other team members all around the world.
+Sourcegraph is an  [all-remote](https://about.sourcegraph.com/handbook/company/remote)  company. This means we all need to make an effort to travel to meet up in person and spend quality time with other team members all around the world.
 
-Team meetups are very important for building cohesiveness in remote teams and spreading Sourcegraph culture across the company.
+Team meetups are very important for building cohesiveness in remote teams and spreading Sourcegraph culture across the company. 
 
-We want and expect you to use your full travel budgets described here each year! For this reason, they aren't transferrable, and unused budget doesn't roll over into the next year.
+Be economical in choosing your accommodations, modes of transportation, and where you’ll be dining. We understand that there may be limits, and we urge you to find options that are both convenient for you while being reasonable reimbursements for the company. We trust you to use your best judgement while spending company money.
```

Comment:
This feels somewhat at odds with the more specific guidance further down around accommodation cost.

Practically speaking, the last place I worked with similar verbiage had problems with people interpreting this kind of advice _really_ differently: a fresh uni grad who's used to backpacking will probably have a very different idea of "economical" than a professional who's been travelling for work for decades, and that's without even getting into cross-cultural differences.

I understand the reasons why we don't just want to say "your budget is $400 per night; go" up front, but we might need more detail here around how to choose.